src/pydata_sphinx_theme/assets/styles/components/_search.scss

```
@@ -65,7 +65,7 @@
   color: var(--pst-color-text-muted);
 
   &:hover {
-    color: var(--pst-color-primary);
+    color: var(--pst-color-secondary);
```

equivalent result, but easier-to-understand source?
```suggestion
    color: var(--pst-color-link-hover);
```